src/pages/programs/index.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+
+import ArticleSpotlightCard from "../../components/grid-aware/ArticleSpotlightCard";
+import CommunityRepProgramBlock from "../../components/grid-aware/CommunityRepProgramBlock";
+import classroom from "../../components/grid-aware/ImageHeader/stories/classroom.png";
+import ProgramsPageLargeParagraph from "../../components/grid-aware/ProgramsPageLargeParagraph";
+import ProgramsSubTitleBlock from "../../components/grid-aware/ProgramsSubTitleBlock";
+import SFServiceGuideBlock from "../../components/grid-aware/SFServiceGuideBlock";
+import ShelterConnectBlock from "../../components/grid-aware/ShelterConnectBlock";
+
+import Layout from "../../components/layout";
+import articleSpotlightImage from "./laura-barerra-vera.png";
+export default () => (
+  <Layout>
+    <ProgramsPageLargeParagraph
+      title="Our Programs"
+      description="ShelterTech's programs are designed to suppport the underserved community experiencing homelessness and housing insecurity through WiFi connectivity and acccess to online resources."
+    />
+    <ProgramsSubTitleBlock title="ShelterTech offers the following programs:" />
+    <SFServiceGuideBlock
```

Comment:
Ah, all three of these blocks (SF Service Guide, ShelterConnect, and Community Rep) should just use the same, single component definition but with different props, since they all have an identical visual structure. Although the main description may have a different number of paragraphs or bulleted lists, we can actually just represent that as a single `description` prop that accepts a JSX fragment, which would allow us to pass in different sets of `<p>`, `<ul>`, and other elements.

You can take a look at how we do this for the ImageHeader component as instantiated in the new [About Us page](https://github.com/ShelterTechSF/sheltertech.org/blob/dfd799813a656f4fd3be125e704a75b943fcbe4b/src/pages/about/index.tsx#L23-L68) and on the [Volunteer page](https://github.com/ShelterTechSF/sheltertech.org/blob/dfd799813a656f4fd3be125e704a75b943fcbe4b/src/pages/volunteer/index.tsx#L40).

Do you think you could go consolidate these into a single component and just provide them different props on the programs page? We could call the component the `ProgramBlock` or something like that.